src/api/ItemsServiceApi.ts

```diff
@@ -0,0 +1,35 @@
+import { config } from '@/constants/envVariables'
+import { BaseServiceInstance } from './BaseServiceApi'
+import { IBaseServiceApi, IItemsServiceApi } from '@/types/serviceTypes'
+import { Item } from '@/types/dataTypes'
+
```

Comment:
class ItemServiceError extends ApiError {
constructor(message: string, code, description) {
super(message, code, description)
   this.message = 'item service error: ${message}'
}
}

const baseServiceInstance = new BaseServiceInstance(url,ItemServiceError )
export const ItemServiceInstance = new ItemsServiceApi(
  baseServiceInstance
)